Description: configure.ac: remove -shared to continue building a static lib
Author: Andreas Barth <aba@ayous.org>

--- meanwhile-1.0.2.orig/configure.ac
+++ meanwhile-1.0.2/configure.ac
@@ -44,7 +44,7 @@ case "${host}" in
 *mingw32 | *cygwin)
 	MW_SO_OS_FLAGS="-shared -no-undefined";;
 *)
-	MW_SO_OS_FLAGS="-shared";;
+	MW_SO_OS_FLAGS="";;
 esac
 AC_SUBST(MW_SO_OS_FLAGS)
 
